boards/catie/zest_core_stm32l4a6rg/zest_core_stm32l4a6rg.dts

```diff
@@ -92,22 +92,18 @@
 };
 
 &timers1 {
+	st,prescaler = <1000>;
```

Comment:
Why do we need to set the prescaler here?

Reply:
It is not needed to be here but if not, the user should have to do it.
By defining the prescaler here, we are setting it to a default value allowing a PWM at millisecond period (16-bits timer).
I have seen other board doing it (cf. @Copper-Bot) and it doesn't prevent the user to change it in the `app.overlay`

Reply:
OK, it's common practice then, we should do it too.

Thanks for the answer!